### PR TITLE
Rename URDNA to RDFC1.0.

### LINKED ISSUE #67: Duplicate test cases
Some test cases have identical inputs (`*-in.nq`) and outputs (`-urdna2015.nq`) to each other.
They can be grouped as follows (for example, [test024-in.nq](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test024-in.nq) is identical to [test025-in.nq](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test025-in.nq); [test024-urdna2015.nq](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test024-urdna2015.nq) is the same as [test025-urdna2015.nq](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test025-urdna2015.nq))

- [024](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test024) ([in](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test024-in.nq)/ [out](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test024-urdna2015.nq)), [025](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test025) ([in](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test025-in.nq) / [out](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test025-urdna2015.nq)), [026](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test026) ([in](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test026-in.nq) / [out](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test026-urdna2015.nq)), [027](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test027) ([in](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test027-in.nq) / [out](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test027-urdna2015.nq)), [028](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test028) ([in](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test028-in.nq) / [out](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test028-urdna2015.nq)), [029](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test029) ([in](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test029-in.nq) / [out](https://github.com/w3c/rdf-canon/blob/4bbba31db202118701af914c73d01335dbb1114f/tests/urdna2015/test029-urdna2015.nq))
- [031](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test031), [032](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test032)
- [033](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test033), [034](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test034)
- [035](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test035), [036](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test036), [037](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test037)
- [040](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test040), [041](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test041), [042](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test042) (fixed; I originally wrongly noted as 041, 042, 043)

It seems they originally came from distinct JSON-LD documents that resulted in the same sorted N-Quads. We can repair some of them to reflect the original intent of each test case. For example, in the case of 024-029:

test024-in.nq (original): "blank node - double circle of 3 (1-2-3)[^1]"
```
_:e0 <http://example.org/vocab#next> _:e1 .
_:e0 <http://example.org/vocab#prev> _:e2 .
_:e1 <http://example.org/vocab#next> _:e2 .
_:e1 <http://example.org/vocab#prev> _:e0 .
_:e2 <http://example.org/vocab#next> _:e0 .
_:e2 <http://example.org/vocab#prev> _:e1 .
```

test025-in.nq (updated): "blank node - double circle of 3 (1-3-2)"
```
_:e0 <http://example.org/vocab#next> _:e2 .
_:e0 <http://example.org/vocab#prev> _:e1 .
_:e2 <http://example.org/vocab#next> _:e1 .
_:e2 <http://example.org/vocab#prev> _:e0 .
_:e1 <http://example.org/vocab#next> _:e0 .
_:e1 <http://example.org/vocab#prev> _:e2 .
```

test026-in.nq (updated): "blank node - double circle of 3 (2-1-3)"
```
_:e1 <http://example.org/vocab#next> _:e0 .
_:e1 <http://example.org/vocab#prev> _:e2 .
_:e0 <http://example.org/vocab#next> _:e2 .
_:e0 <http://example.org/vocab#prev> _:e1 .
_:e2 <http://example.org/vocab#next> _:e1 .
_:e2 <http://example.org/vocab#prev> _:e0 .
```
(027-029 are omitted)

I checked that my implementation [passed](https://github.com/yamdan/rdf-canon-rust/commit/85c9d4af7e4b5c47f10f229f9637e1468fe0e5e0) the updated test024-029.

If I can confirm this is not my misunderstanding, I would like to create a PR that includes other test cases.

[^1]: from the description in [024](https://w3c.github.io/rdf-canon/tests/#manifest-urdna2015#test024), which should be renamed as "0-1-2" to be more understandable.

### LINKED ISSUE #68: Check against i18n Review Checklist
This short review is for the following spec: [RDF Dataset Canonicalization](https://www.w3.org/TR/rdf-canon/).

1. [ ] _If the spec (or its implementation) contains any natural language text that will be read by a human (this includes error messages or other UI text, JSON strings, etc, etc),_ **ensure that there’s metadata about and support for basic things such as language and text direction**.  Also check the detailed guidance for [Language](https://www.w3.org/TR/international-specs/#resource) and [Text direction](https://www.w3.org/TR/international-specs/#text_direction).

    Comments_go_here.
    - [ ] RDF Datasets may contain natural language text. The methods for encoding such text are described in existing RDF specs. See the [Turtle specification on RDF Literals](https://www.w3.org/TR/2014/REC-turtle-20140225/#literals) for example. The RDF Canonicalization spec accepts any serialization of RDF but does not create a new one. The majority of the c14n spec is concerned with the algorithm for labeling blank nodes and ordering the resulting quads to create a canonical form. At the time of writing, the WG has not settled on whether such a canonical form has a canonical serialization (although it seems likely that it will). In other words, the current working assumption is that i18n issues are covered by the existing RDF standards and that no new issues are created by canonicalization. It is on this point that we would be very grateful for feedback.
    
2. [ ] _If the spec (or its implementation) allows content authors to produce typographically appealing text, either in its own right, or in association with graphics._ **take into account the different typographic styles used around the world (for things such as line-breaking, text justification, emphasis or other text decorations, text selection and units, etc.)**  Also check the detailed guidance for [Typographic support](https://www.w3.org/TR/international-specs/#typography).

    Comments_go_here.
    - [ ] Not applicable

3. [ ] _If the spec (or its implementation) allows the user to point into text, creates text fragments, concatenates text, allows the user to select or step through text (using a cursor or other methods), etc._ **make allowances for the ways different scripts handle units of text**.  Also check the detailed guidance for [Text-processing](https://www.w3.org/TR/international-specs/#operations).

    Comments_go_here.
    - [ ] Not applicable

4. [ ] _If the spec (or its implementation) allows searching or matching of text, including syntax and identifiers_ **understand the implications of normalisation, case folding, etc**.  Also check the detailed guidance for [Text-processing](https://www.w3.org/TR/international-specs/#operations).

    Comments_go_here
    - [ ] This is where there *may* be a need for new i18n considerations over and above those already defined in the RDF specifications but we think it unlikely.

5. [ ] _If the spec (or its implementation) sorts text_ **ensure that it does so in locally relevant ways**. Also check the detailed guidance for [Text-processing](https://www.w3.org/TR/international-specs/#operations).

    Comments go here.
    - [ ] We believe the existing RDF specs offer sufficient clarity in this regard but, again, we'd be grateful for any feedback on this. The RDF Dataset c14n algorithm *does* include a step where all quads are arranged in lexical order.

6. [ ] _If the spec (or its implementation) captures user input_ **ensure that it also captures metadata about language and text direction, and that it accommodates locale-specific input methods**. 

    Comments go here.
    - [ ] Not applicable

7. [ ] _If the spec (or its implementation) deals with time in any way that will be read by humans and/or crosses time zone boundaries_  **ensure that it will represent time as expected in locales around the world, and manage the relationship between local and global/absolute time**. Also check the detailed guidance for [Local dates, times and formats](https://www.w3.org/TR/international-specs/#locale).

    Comments go here.
    - [ ] Not applicable

8. [ ] _If the spec (or its implementation) allows any character encoding other than UTF-8._  **make sure you have a convincing argument as to why, and then ensure that the character encoding model is correct**.   Also check the detailed guidance for [Characters](https://www.w3.org/TR/international-specs/#characters).

    Comments go here.
    - [ ] Not applicable

9. [ ] _If the spec (or its implementation) defines markup_  **ensure support for internationalisation features and avoid putting human-readable text in attribute values or plain-text elements**.  Also check the detailed guidance for [Markup & syntax](https://www.w3.org/TR/international-specs/#markup).

    Comments go here.
    - [ ] Not applicable

10. [ ] _If the spec (or its implementation) deals with names, addresses, time & date formats, etc_  **ensure that the model is flexible enough to cope with wide variations in format, levels of data, etc**.   Also check the detailed guidance for [Local dates, times and formats](https://www.w3.org/TR/international-specs/#locale).

    Comments go here.
    - [ ] Not applicable

11. [ ] _If the spec (or its implementation) describes a format or data that is likely to need localization._  **ensure that there’s an approach in place which allows effective storage and labelling of, and access to localised alternatives for strings, text, images, etc**.  

    Comments go here.
    - [ ] We believe this is covered by the base RDF standards and won't be affected by c14n.

12. [ ] _If the spec (or its implementation) makes any reference to or relies on any cultural norms_  **ensure that it can be adapted to suit different cultural norms around the world (ranging from depictions of people or gestures, to expectations about gender roles, to approaches to work and life, etc)**.

    Comments go here.
    - [ ] Not applicable

Short i18n review checklist is [here](https://www.w3.org/International/i18n-drafts/techniques/shortchecklist.html)

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes #67. Fixes #68.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/116.html" title="Last updated on Jun 9, 2023, 12:03 AM UTC (df11918)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/116/e98a348...df11918.html" title="Last updated on Jun 9, 2023, 12:03 AM UTC (df11918)">Diff</a>

### CODE REVIEW

**Review by yamdan — COMMENTED** [review]
Thank you @gkellogg . I just added a few suggestion, but please feel free to correct them if my English is unnatural.

File: spec/index.html [hunk 1]
```diff
@@ -3300,6 +3304,20 @@ <h2>A Canonical form of N-Quads</h2>
...
+    should be compatibile with this specification.
+    The minor change is in the <a>canonical n-quads form</a> where
+    some control characters were previously represented without escaping.
+    The version of the algorithm defined in <a href="#canonical-quads" class="sectionRef"></a>
+    clarifies the representation of <a data-cite="RDF11-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a> and the characters
+    within <a data-cite="N-QUADS#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>
+    that are encoded using <code><a data-cite="N-QUADS#grammar-production-ECHAR">ECHAR</a></code>.</p>
+</section>
+
 <section id="urgna2021" class="appendix informative algorithm">
```
[the remark this answers is not in the document]
Comment (gkellogg) [reply spec/index.html:3321]: Not at this point.

**Review by yamdan — APPROVED** [review]

**Review by iherman — APPROVED** [review]
I believe this change should be explicitly noted in the "Changes" appendix.

File: spec/index.html [hunk 2]
```diff
@@ -3300,7 +3304,24 @@ <h2>A Canonical form of N-Quads</h2>
...
-<section id="urgna2021" class="appendix informative algorithm">
+<section id="urdna2015" class="appendix informative">
+  <h2>URDNA2015</h2>
+
+  <p>[[[CCG-RDC-FINAL]]] [[CCG-RDC-FINAL]] describes
+    "Universal RDF Dataset Normalization Algorithm 2015"
+    (<abbr title="Universal RDF Dataset Normalization Algorithm 2015"><dfn class="export">URDNA2015</dfn></abbr>),
+    essentially the same algorithm    
+    as <a>RDFC-1.0</a>, and generally implementations implementing URDNA2015
+    should be compatibile with this specification.
```
Comment (dlongley) [inline spec/index.html:3315]:
```suggestion
    should be compatible with this specification.
```

**Review by dlongley — APPROVED** [review]
Thanks!

**Pull request merged**